README.md

```diff
@@ -133,6 +135,21 @@ private void OnFail(RESTConnector.Error error, Dictionary<string, object> custom
 }
 ```
 
+### Authentication for Visual Recognition
+Note there are different endpoints for authenticating a Visual Recognition instance depending on if you are authenticating using IAM or apikey. 
```

Comment:
Thinking out loud here, this can be confusing for VR actually because we used to have an `api_key` pre-IAM days and now we have `apikey` in credentials. But the point is that `apikey` word can confuse users.

Reply:
Ah good catch! This needs to be removed since we handle this internally based on if they authenticate using apikey or tokenOptions